src/openforms/variables/models.py

```diff
@@ -145,7 +145,7 @@ def request_arguments(self, context: DataMapping) -> dict:
         # this catches faults requests doesn't: https://github.com/psf/requests/issues/6359
         HeaderValidator()(headers)
 
-        escaped_for_path = {k: quote_plus(str(v)) for k, v in context.items()}
+        escaped_for_path = {k: quote(str(v), safe="") for k, v in context.items()}
```

Comment:
removed the default `safe="/"` argument - in our case it is definitely unsafe. the plus does not need to be quoted, as that's only relevant in form data apparently.